src/Tickets/Seating/Service/Maps.php

```diff
@@ -184,4 +189,70 @@ public static function update_transient_name(): string {
 	public static function update_transient_expiration() {
 		return 12 * HOUR_IN_SECONDS;
 	}
+	
+	/**
+	 * Checks if the map has layouts.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $map_id The ID of the map.
+	 *
+	 * @return bool The number of layouts.
+	 */
+	public function map_has_layouts( string $map_id ): bool {
```

Comment:
Add test coverage, either directly or indirectly (e.g. testing the `Ajax` controller)